.github/workflows/release-update-stable-tag.yml

```diff
@@ -0,0 +1,268 @@
+                const bPart = bParts[i] || 0;
+
+                if (aPart > bPart) return 1;
+                if (aPart < bPart) return -1;
+              }
+              return 0;
+            };
+
+            const currentStable = '${{ steps.current-stable.outputs.current-stable }}';
+            const targetVersion = '${{ steps.extract-version.outputs.version }}';
+            const allowRevert = '${{ github.event.inputs.allow-revert }}' === 'true';
+
+            console.log(`Current stable: ${currentStable}`);
+            console.log(`Target version: ${targetVersion}`);
+
+            if (currentStable === targetVersion) {
+              core.setFailed(`Stable tag is already set to ${targetVersion}. No update needed.`);
+              return;
+            }
+
+            const comparison = compareVersions(targetVersion, currentStable);
+
+            if (comparison > 0) {
+              console.log(`✅ Normal upgrade - updating from '${currentStable}' to '${targetVersion}'`);
+              return;
+            }
+
+            console.log(`⚠️ Warning: Target version (${targetVersion}) is lower than current stable tag (${currentStable})`);
+            if (!allowRevert) {
+              core.setFailed('Cannot set stable tag to a lower version without explicit revert confirmation. Please check the "Allow revert" checkbox if this is intentional.');
+              return;
+            }
+
+            console.log(`✅ Revert confirmed - proceeding with downgrade from '${currentStable}' to '${targetVersion}'`);
+
+  update-wporg-stable-tag:
+    runs-on: ubuntu-latest
+    needs: validate-release
+    steps:
+      - name: Install SVN
+        run: sudo apt-get install -y subversion
+
+      - name: Update SVN stable tag
+        env:
+          TARGET_VERSION: ${{ needs.validate-release.outputs.version }}
+        run: |
+          echo "🔄 Checking out SVN trunk..."
+          svn checkout "${{ secrets.WPORG_SVN_URL }}/trunk" ./svn-trunk --depth files --username "${{ secrets.WPORG_SVN_USERNAME }}" --password "${{ secrets.WPORG_SVN_PASSWORD }}" --non-interactive
+          cd ./svn-trunk
+
+          README_PATH="./readme.txt"
+
+          if [ ! -f "$README_PATH" ]; then
+            echo "::error::readme.txt not found at $README_PATH"
+            exit 1
+          fi
+
+          echo "📋 Current stable tag line:"
+          grep "^Stable tag:" "$README_PATH" || echo "::error::Stable tag line not found"
+          echo "=========================="
+
+          # Update the stable tag
+          sed -i "s/^Stable tag: .*/Stable tag: $TARGET_VERSION/" "$README_PATH"
+
+          echo "📝 Updated stable tag line:"
+          grep "^Stable tag:" "$README_PATH" || echo "::error::Stable tag line not found"
+          echo "=========================="
+
+          # Check if there are any changes
+          if svn diff --quiet; then
+            echo "ℹ️ No changes detected - stable tag is already $TARGET_VERSION"
```

Comment:
Shouldn't this trigger an error? We've validated the version before so if this happens now something weird happened while the workflow was running.